src/resolvers/generate.ts

```
@@ -0,0 +1,740 @@
+                      }
+
+                      if (field.list && field.kind === 'enum') {
+                        writer
+                          .write(`if (Array.isArray(normalizedInput.${field.name})) `)
+                          .inlineBlock(() => {
+                            writer.writeLine(
+                              `normalizedInput.${field.name} = \`{\${(normalizedInput.${field.name} as string[]).join(',')}}\`;`
+                            );
+                          })
+                          .blankLine();
+                        continue;
```

Biome

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)